config/routes.rb

```diff
@@ -23,7 +23,6 @@ def scoped_navigation_routes(context, navigation, as_redirects: false)
   # All routes in this scope will be prefixed with /locale if an available locale is set. See default_url_options in
   # application_controller.rb and http://guides.rubyonrails.org/i18n.html for more info on this approach.
   scope '(:locale)', locale: /#{I18n.available_locales.join('|')}/ do
-    get '/:locale' => 'public_pages#home'
```

Comment:
so this seems like it's been a bug for a long time, probably since i did the first pass i18n routes. this resulted in e.g. url_for /en outputting "/en/en". you can see this in the open graph and twitter meta tags in production currently. removing this line fixes that and i don't think it was doing anything good that we'll miss.